Homework_1/Task_2.py

```diff
@@ -0,0 +1,22 @@
+#Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+#Дано a, b, c - стороны предполагаемого треугольника. 
+#Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+#Если хотя бы в одном случае отрезок окажется больше суммы двух других, 
+#то треугольника с такими сторонами не существует. 
+#Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+def is_valid_triangle(side1, side2, side3):
+    if side1 == side2 == side3:
+        print("Треугоьник равносторонний")
+    elif side1 == side2 or side1 == side3 or side2 == side3:
+        print("Треугольник равнобедренный")
+    else: print("Треугольник разносторонний")
```

Comment:
Такая запись хоть и допустима, но не принята, согласно документации PEP8.
```suggestion
    else:
        print("Треугольник разносторонний")
```